package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "scripts": {
+    "start": "webpack-dev-server --hot --inline --content-base static/",
+    "build": "rimraf dist && webpack",
+    "serve": "static dist",
+    "watch": "webpack --watch"
+  },
+  "devDependencies": {
```

Comment:
I don't see any crufty things in here besides bootstrap. I'll go through these dependencies if / when we decide what to do with bootstrap.